src/transforms/normalize.d.ts

```diff
@@ -22,12 +22,73 @@ export interface NormalizeOptions {
   basis?: NormalizeBasis;
 }
 
+/**
+ * Normalize series values relative to the given basis. For example, if the
```

Comment:
This is another instance where we want the first sentence to emphasize what is unique about normalizeX as opposed to shared across all variations of the normalize transform: it should mention specifically which channels are going to be transformed. (Fixing as I edit.)

Reply:
Also, since the normalize transform is built on top of the map transform, we should also mention that data are first grouped into series. From Plot.map in the README:

> Groups on the first channel of z, fill, or stroke, if any, and then for each channel declared in the specified outputs object, applies the corresponding map method.